src/runtime/pymodule.cs

```diff
@@ -29,6 +32,13 @@ public PyModule Reload()
             return new PyModule(ref op);
         }
 
+        public void AddToSysModules()
+        {
+            BorrowedReference modules = Runtime.PyImport_GetModuleDict();
+            int rc = Runtime.PyDict_SetItemString(modules, this.Name, this.Reference);
+            PythonException.ThrowIfIsNotZero(rc);
+        }
+
```

Comment:
Instead of adding potentially even more methods, that have to do with `sys.modules` in `PyModule`, expose a static member `PyModule.SysModules`, and let users do checking and adding on their own.

It should replace `AddToSysModules` and `IsInSysModules`.

Reply:
BorrowedReference is an internal struct, what is the best thing to return then since this would be a public API?

Reply:
For `SysModules`? `PyDict` - it is a dictionary.

Reply:
Sorry, missed the PyDict in your original comment.